### PR TITLE
1209khs

### PR DESCRIPTION
**Pull request by KHS-kr**

### CODE REVIEW

File: routes/breakdown.js [hunk 1]
```diff
@@ -13,10 +13,24 @@ masterCheck = async function(userId, eventId) {
...
+    return check !== null;
+};
+
 router.get('/event/:eventId', async function(req, res, next) {
     const responseJson = {};
     const eventId = req.params.eventId;
-    if(!await masterCheck(res.locals.user.id, eventId)) {
+    const isMaster = !await masterCheck(res.locals.user.id, eventId);
+    const isAdmin = !await adminCheck(res.locals.user.id, eventId);
+    if(isMaster && isAdmin) {
```
Comment (ans2552) [inline routes/breakdown.js]: isMaster = await masterCheck(res.locals.user.id, eventId);
isAdmin = await adminCheck(res.locals.user.id, eventId);
!isMaster && !isAdmin
문맥상 이게 맞는듯

File: routes/event.js [hunk 2]
```diff
@@ -27,32 +27,48 @@ adminCheck = async function(userId, eventId) {
     return check !== null;
 };
 
-// post1 : check phonenubmer
-router.post('/', async function(req, res, next) {
-    const responseJson = {};
-    const eventAdmin = req.body.eventAdmin;
+getUserByPhone = async function(phone) {
```
Comment (ans2552) [inline routes/event.js]: phones가 더 좋앗을거같으는낌

File: routes/event.js [hunk 3]
```diff
@@ -207,6 +242,61 @@ router.put('/:id', async function(req, res, next) {
...
+                                attributes: ['firebase_token'],
+                            },
+                        );
+
+                        fcmList.push(result3.firebase_token);
+                    }
+                }
+            }
+
+            const result4 = await Event.findOne(
```
Comment (ans2552) [inline routes/event.js]: body.title에 있기때문에 findone안해도될듯?

File: routes/breakdown.js (hunk 1 above)
Comment (KHS-kr) [reply routes/breakdown.js]: 쳌

File: routes/event.js (hunk 2 above)
Comment (KHS-kr) [reply routes/event.js]: getUsersByPhones, phones

File: routes/event.js (hunk 3 above)
Comment (KHS-kr) [reply routes/event.js]: 요거 순간 바뀌기전께 들어오는걸로 착각햇음
쳌

File: routes/event.js [hunk 4]
```diff
@@ -104,26 +138,15 @@ router.post('/', async function(req, res, next) {
         res.locals.urAdmin = urAdmin;
         next();
     }
+
```
Comment (ans2552) [inline routes/event.js]: delete

File: routes/event.js [hunk 5]
```diff
@@ -165,47 +188,47 @@ router.post('/', async function(req, res, next) {
...
                 {
                     where: {
                         id: pEventId,
                     },
-                    transaction
+                    transaction,
                 },
             );
-            console.log(myAdmin)
+            console.log(myAdmin);
```
Comment (ans2552) [inline routes/event.js]: delete

File: routes/event.js (hunk 4 above)
Comment (KHS-kr) [reply routes/event.js]: check

File: routes/event.js (hunk 5 above)
Comment (KHS-kr) [reply routes/event.js]: check

**Pull request merged**